src/machine.js

```
@@ -44,8 +44,16 @@ class Machine {
   }
 
   getEdgeFromTransition(transition: TransitionObject): Edge {
+    if (!this.#state) {
+      this.log('Invalid State - unable get Edges for Transistion');
+      return false;
+    }
+    if (!transition) {
+      this.log('Invalid Transition parameter passed');
+      return false;
```

this breaks the expected return- you should throw an error